test/fixtures/data/apis/api3/deployments/2022-04-17/airnode/gcp/secrets.env

```diff
@@ -1,2 +1,3 @@
 AIRNODE_WALLET_MNEMONIC=
-HTTP_SIGNED_DATA_GATEWAY_KEY_API3_AWS=
\ No newline at end of file
+HTTP_SIGNED_DATA_GATEWAY_KEY_API3_AWS=
+GCP_PROJECT_ID=
```

Comment:
I thought by [this](https://github.com/api3dao/operations/pull/52#discussion_r882665100) comment that GCP secrets.evn was supposed to be a json file. I might be missing something right?

Reply:
the `gcp` file is supposed to be a json file. In AWS case the `aws` file is supposed to be a .env file . 
But I identified a major issue with secrets naming when seeing this comment so I will push a fix